File: cpp/23.cpp
#include <iostream>

int main()
{
    int conj1[5] = {7, 25, -2, 80, -31};
    int conj2[5] = {81, 12, 7, -90, 16};
    int conj3[5];

    for (int i = 0; i < 5; ++i)
    {
        conj3[i] = conj1[i] + conj2[i];
        std::cout << conj3[i] << std::endl;
    }

    return 0;
}

File: Lua/23.lua
local conj1 = {7, 25, -2, 80, -31}
local conj2 = {81, 12, 7, -90, 16}
local conj3 = {}

for i = 1, #conj1 do
    conj3[i] = conj1[i] + conj2[i]
    print(conj3[i])
end